charts/nri-bundle/requirements.lock

```diff
@@ -16,15 +16,15 @@ dependencies:
   version: 1.9.5
 - name: newrelic-logging
   repository: file://../newrelic-logging
-  version: 1.5.0
+  version: 1.6.0
 - name: newrelic-pixie
   repository: file://../newrelic-pixie
-  version: 1.1.0
-- name: pixie-chart-helm2
-  repository: https://pixie-chart-helm2.storage.googleapis.com
-  version: 0.8.2
+  version: 1.1.1
+- name: pixie-operator-helm2-chart
+  repository: https://pixie-operator-charts.storage.googleapis.com
+  version: 0.0.4
```

Comment:
Can we actually update this to 0.0.5? We just did an operator release today, so it would be good to pull in that latest version. Thanks!

Reply:
Done in https://github.com/newrelic/helm-charts/pull/467/commits/17604c77b57c50d542055e1136cf20b7f07056ee :)